packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -188,6 +188,11 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return this._containerRuntime.disableIsolatedChannels;
     }
 
+    /** We should GC blobs in the summary only if the container runtime is not writing the GC tree. */
+    protected get shouldWriteGCBlob(): boolean {
```

Comment:
Same here - even more important to get names write, as the meaning for shouldWriteGCTree & shouldWriteGCBlob are reversed. And both functions are in the same layer (even though on different objects)

Reply:
Actually, if you use writeDataAtRoot terminology across layers, then I think it's much clearer on intentions (and same term used across layers)

Reply:
Updated it to `writeGCDataAtRoot`